README.md

```diff
@@ -1,3 +1,15 @@
-# @web3modal/react-native
+### 📚 [Documentation](https://docs.walletconnect.com/2.0/reactnative/web3modal/Installation)
 
-Web3Modal SDK for React-Native
+### 🔎 [Examples](https://github.com/WalletConnect/react-native-examples/tree/main/dapps/v2Explorer)
+
+### 🧪 [Laboratory](https://lab.web3modal.com)
+
+### 🔗 [Website](https://web3modal.com)
+
+# Web3Modal SDK for React Native
+
+Your on-ramp to web3 multichain. Web3Modal is a versatile library that makes it super easy to connect users with your Dapp and start interacting with the blockchain.
```

Comment:
Nitpick: could move the text about `Web3Modal SDK for React Native` to the top of the README so people know what it is from initial read.